VisualPinball.Unity/VisualPinball.Unity.Editor/Inspectors/HitTargetInspector.cs

```diff
@@ -1,82 +1,108 @@
-﻿using UnityEditor;
-using VisualPinball.Unity.VPT.HitTarget;
-
-namespace VisualPinball.Unity.Editor.Inspectors
-{
-	[CustomEditor(typeof(HitTargetBehavior))]
-	public class HitTargetInspector : ItemInspector
-	{
-		private HitTargetBehavior _target;
-		private bool _foldoutColorsAndFormatting = true;
-		private bool _foldoutPosition = true;
-		private bool _foldoutPhysics = true;
-		private bool _foldoutMisc = true;
-
-		protected override void OnEnable()
-		{
-			base.OnEnable();
-			_target = target as HitTargetBehavior;
-		}
-
-		public override void OnInspectorGUI()
-		{
+﻿using UnityEditor;
+using VisualPinball.Engine.VPT;
+using VisualPinball.Unity.VPT.HitTarget;
+
+namespace VisualPinball.Unity.Editor.Inspectors
+{
+	[CustomEditor(typeof(HitTargetBehavior))]
+	public class HitTargetInspector : ItemInspector
+	{
+		private HitTargetBehavior _target;
+		private bool _foldoutColorsAndFormatting = true;
+		private bool _foldoutPosition = true;
+		private bool _foldoutPhysics = true;
+		private bool _foldoutMisc = true;
+
+		private static string[] _targetTypeStrings = {
+			"DropTargetBeveled",
+			"DropTargetSimple",
+			"HitTargetRound",
+			"HitTargetRectangle",
+			"HitFatTargetRectangle",
+			"HitFatTargetSquare",
+			"DropTargetFlatSimple",
+			"HitFatTargetSlim",
+			"HitTargetSlim",

```

Comment:
Those are exclusively used for the dropdown, right? Can we name them less cody? Like "Drop Target: Beveled", "Drop Target: Simple", etc?

Reply:
Same for gates, kickers, light states, ramps and triggers.